Makefile.buildx.mk

```diff
@@ -0,0 +1,178 @@
+    export PLATFORMS="local"
+else ifeq (${IMG_RESULT}, cache)
+	# if cache is specified, image will only be available in the build cache, it won't be pushed or loaded
+	# therefore no PUSH_ARG will be specified
+else
+	export PUSH_ARG="--push"
+endif
+
+# Name of the multiarch image for cspc-operator
+DOCKERX_IMAGE_CSPC_OPERATOR:=${IMAGE_ORG}/cspc-operator:${TAG}
+
+# Name of the multiarch image for cvc-operator
+DOCKERX_IMAGE_CVC_OPERATOR:=${IMAGE_ORG}/cvc-operator:${TAG}
+
+# Name of the multiarch image for pool-manager
+DOCKERX_IMAGE_POOL_MANAGER:=${IMAGE_ORG}/pool-manager:${TAG}
+
+# Name of the multiarch image for volume-manager
+DOCKERX_IMAGE_VOLUME_MANAGER:=${IMAGE_ORG}/volume-manager:${TAG}
+
+# Name of the multiarch image for cstor-webhook
+DOCKERX_IMAGE_CSTOR_WEBHOOK:=${IMAGE_ORG}/cstor-webhook:${TAG}
+
+
+# Build cstor-operator docker images with buildx
+# Experimental docker feature to build cross platform multi-architecture docker images
+# https://docs.docker.com/buildx/working-with-buildx/
+.PHONY: buildx.cspc-operator
+buildx.cspc-operator: bootstrap clean 
+	@echo '--> Building cspc-operator binary...'
+	@pwd
+	@PNAME=${CSPC_OPERATOR} CTLNAME=${CSPC_OPERATOR} BUILDX=true sh -c "'$(PWD)/build/build.sh'"
+	@echo '--> Built binary.'
+	@echo
+
+.PHONY: docker.buildx.cspc-operator
+docker.buildx.cspc-operator:
+	export DOCKER_CLI_EXPERIMENTAL=enabled
+	@if ! docker buildx ls | grep -q container-builder; then\
+		docker buildx create --platform ${PLATFORMS} --name container-builder --use;\
+	fi
+	@docker buildx build --platform ${PLATFORMS} \
+		-t "$(DOCKERX_IMAGE_CSPC_OPERATOR)" ${DBUILD_ARGS} -f cspc-operator.Dockerfile \
+		. ${PUSH_ARG}
+	@echo "--> Build docker image: $(DOCKERX_IMAGE_CSPC_OPERATOR)"
+	@echo
+
+.PHONY: buildx.cvc-operator
+buildx.cvc-operator: bootstrap clean 
+	@echo '--> Building cvc-operator binary...'
+	@pwd
+	@PNAME=${CVC_OPERATOR} CTLNAME=${CVC_OPERATOR} BUILDX=true sh -c "'$(PWD)/build/build.sh'"
+	@echo '--> Built binary.'
+	@echo
+
+.PHONY: docker.buildx.cvc-operator
+docker.buildx.cvc-operator:
+	export DOCKER_CLI_EXPERIMENTAL=enabled
+	@if ! docker buildx ls | grep -q container-builder; then\
+		docker buildx create --platform ${PLATFORMS} --name container-builder --use;\
+	fi
+	@docker buildx build --platform ${PLATFORMS} \
+		-t "$(DOCKERX_IMAGE_CVC_OPERATOR)" ${DBUILD_ARGS} -f cvc-operator.Dockerfile \
+		. ${PUSH_ARG}
+	@echo "--> Build docker image: $(DOCKERX_IMAGE_CVC_OPERATOR)"
+	@echo
+
+.PHONY: buildx.pool-manager
+buildx.pool-manager: bootstrap clean 
+	@echo '--> Building pool-manager binary...'
+	@pwd
+	@PNAME=${POOL_MANAGER} CTLNAME=${POOL_MANAGER} BUILDX=true sh -c "'$(PWD)/build/build.sh'"
+	@echo '--> Built binary.'
+	@echo
+
+.PHONY: docker.buildx.pool-manager
+docker.buildx.pool-manager:
+	export DOCKER_CLI_EXPERIMENTAL=enabled
+	@if ! docker buildx ls | grep -q container-builder; then\
+		docker buildx create --platform ${PLATFORMS} --name container-builder --use;\
+	fi
```

Comment:
Seeing as we're using this logic everywhere perhaps we can have a single target that could be used to ensure this is set up instead of having it duplicated?